Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -25,6 +25,7 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
 
     private var completionActive = false
     private var canAutocomplete = true
+    private var placeCaretAtEnd = false
```

Comment:
Also, a comment above this variable describing the delicate interaction between `highlightAll()`, `touchesBegan`, and `touchesEnded` would be useful. I'd just give a small description about why we need this variable.